common/src/main/java/org/apache/sedona/common/raster/RasterConstructors.java

```diff
@@ -157,7 +157,7 @@ private static int getDataTypeCode(String s) {
                 return 0;
             case "f":
                 return 4;
-            case "u":
```

Comment:
Do you need to update the MakeEmptyRaster doc?

Reply:
No the doc is correct, there was a bug in the code